app/models/profile.rb

```diff
@@ -1,3 +1,6 @@
 class Profile < ApplicationRecord
   belongs_to :user
+  validates :first_name, presence: true, allow_nil: true
+  validates :last_name, presence: true, allow_nil: true
+  validates :address, presence: true, allow_nil: true
```

Comment:
great ! We add a task on thello with Peter this morning not to forget the validation, but much better to do it during the form creation as you did here !